src/Command.ts

```diff
@@ -574,10 +574,20 @@ export default abstract class Command {
       .replace(/([^\\])\\n/g, '$1\\\\\\n');
   }
 
-  public getCsvOutput(logStatement: any[]): string {
+  public getCsvOutput(logStatement: any[], options: GlobalOptions): string {
     const { stringify } = require('csv-stringify/sync');
     const cli = Cli.getInstance();
 
+    if (logStatement && logStatement.length > 0 && !options.query) {
```

Comment:
It seems excluding nested properties does not work, If I run this, I get the properties object in the CSV. That should only happen when using `query` and selecting properties.

```sh
 m365 pa app list --output csv > powerapps.csv
```

Reply:
Can you clarify why you are getting nested objects back? Have run the same command and don't get it back
![image](https://user-images.githubusercontent.com/35696168/229116198-56d357cc-6432-4ba2-85ac-c645f38e0f2e.png)

Reply:
That's odd, I'm now seeing the same. I must have done something wrong!